### PR TITLE
IDMP-243 - stereochemisty

### PR DESCRIPTION
**Pull request by tw-osthus**
added definitions for 
cis, trans, absolute, achiral and mixed

mixed needs clarification with IDMP, the usage is not clear from the TS

### CODE REVIEW

**Review by ElisaKendall — DISMISSED** [review]
This is fine with the updates made to correct hygiene related issues and move equivalences back to the deprecated classes they belong on, eliminate the default IRI Protege adds, etc.

File: ISO/ISO11238-Substances.rdf [hunk 1]
```diff
@@ -995,7 +998,8 @@
 		<rdf:type rdf:resource="&idmp-sub;Stereochemistry"/>
 		<rdfs:label>stereochemistry - meso</rdfs:label>
 		<dct:source>ISO/TS 19844:2018(E) Health informatics - Identification of medicinal products (IDMP) - Implementation guidelines for ISO 11238 for data elements and structures for the unique identification and exchange of regulated information on substances, clause 6.7.1, Table 53, clause B.6.5.1</dct:source>
-		<skos:definition>stereochemistry classifier indicating ...</skos:definition>
+		<skos:definition>stereochemistry classifier indicating that a molecule is a non-optically active member of a set of stereoisomers,at least two of which are optically active</skos:definition>
+		<cmns-av:explanatoryNote>despite containing two or more stereocenters the molecule is not chiral (achiral)</cmns-av:explanatoryNote>
```
Comment (mereolog) [inline ISO/ISO11238-Substances.rdf:1002]: Since this is a sentence, shouldn't the first letter of the first word be UPPERCASE?

**Comment by tw-osthus** [comment]
I can do the merge, but I am concerned that I break things again on commit. I do not have this signing facility for example.

File: ISO/ISO11238-Substances.rdf [hunk 2]
```diff
@@ -913,7 +913,10 @@
 		<rdf:type rdf:resource="&idmp-sub;Stereochemistry"/>
 		<rdfs:label>stereochemistry - absolute</rdfs:label>
 		<dct:source>ISO/TS 19844:2018(E) Health informatics - Identification of medicinal products (IDMP) - Implementation guidelines for ISO 11238 for data elements and structures for the unique identification and exchange of regulated information on substances, clause 6.7.1, Table 53</dct:source>
-		<skos:definition>stereochemistry classifier indicating ...</skos:definition>
+		<skos:definition>stereochemistry classifier indicating that the stereochemical description fully describes of the spatial arrangement of the atom of a chiral molecular entity (or group)</skos:definition>
+		<cmns-av:adaptedFrom>https://en.wikipedia.org/wiki/Absolute_configuration</cmns-av:adaptedFrom>
+		<cmns-av:explanatoryNote>the most common labeling method uses the descriptors R or S and is based on the Cahn-Ingold-Prelog priority rules</cmns-av:explanatoryNote>
+		<cmns-av:explanatoryNote>until 1951 it was not possible to determine the absolute configuration and configuration were indicated relative to glyceraldehyde with the D/L system</cmns-av:explanatoryNote>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply ISO/ISO11238-Substances.rdf:921]: @merelog I'll fix this as well.

**Comment by ElisaKendall** [comment]
> @tw-osthus could your resolve the conflicts that emerged?

@tw-osthus @mereolog I think this is done.

File: ISO/ISO11238-Substances.rdf [hunk 3]
```diff
@@ -915,7 +914,10 @@
 		<rdf:type rdf:resource="&idmp-sub;Stereochemistry"/>
 		<rdfs:label>stereochemistry - absolute</rdfs:label>
 		<dct:source>ISO/TS 19844:2018(E) Health informatics - Identification of medicinal products (IDMP) - Implementation guidelines for ISO 11238 for data elements and structures for the unique identification and exchange of regulated information on substances, clause 6.7.1, Table 53</dct:source>
-		<skos:definition>stereochemistry classifier indicating ...</skos:definition>
+		<skos:definition>stereochemistry classifier indicating that the stereochemical description fully describes of the spatial arrangement of the atom of a chiral molecular entity (or group)</skos:definition>
```
Comment (Heiner-Oberkampf) [inline ISO/ISO11238-Substances.rdf:917]: Grammar: should be ... "describes thespacial..." without "of"

File: ISO/ISO11238-Substances.rdf [hunk 4]
```diff
@@ -976,7 +978,7 @@
 		<rdf:type rdf:resource="&idmp-sub;Stereochemistry"/>
 		<rdfs:label>stereochemistry - cis</rdfs:label>
 		<dct:source>ISO/TS 19844:2018(E) Health informatics - Identification of medicinal products (IDMP) - Implementation guidelines for ISO 11238 for data elements and structures for the unique identification and exchange of regulated information on substances, clause 6.7.1, Table 53, clause B.6.5.1</dct:source>
-		<skos:definition>stereochemistry classifier indicating ...</skos:definition>
+		<skos:definition>stereochemistry classifier indicating that the functional groups (substituents) are on the same side of some plane</skos:definition>
 		<cmns-col:isMemberOf rdf:resource="&idmp-sub;ISO19844-CodeSet"/>
```
Comment (Heiner-Oberkampf) [inline ISO/ISO11238-Substances.rdf:982]: Based on the textual definition we could add an axiom that CIS and TRANS are different namedIndividuals analog to CHIRAL ACHIRAL example.

**Review by Heiner-Oberkampf — CHANGES_REQUESTED** [review]
Small wording correction needed and request to add axiom to make CIS and TRANS stereochemistries distinct.

**Review by mereolog — DISMISSED** [review]

File: ISO/ISO11238-Substances.rdf (hunk 4 above)
Comment (ElisaKendall) [reply ISO/ISO11238-Substances.rdf:985]: Done.

**Review by ElisaKendall — APPROVED** [review]
This addresses the recent feedback, is logically consistent, and passes hygiene.

**Review by mereolog — APPROVED** [review]

**Review by Heiner-Oberkampf — APPROVED** [review]

**Pull request merged**